### PR TITLE
Updates to Profile and Sponsors

### PR DESCRIPTION
**Pull request by jmm8046**
*removing ec and business options on profile
*Updating sponsors
*Updating MLH flag

### CODE REVIEW

File: user-registration-app/src/app/views/user-registration-view/user-registration-view.component.html [hunk 1]
```diff
@@ -30,7 +32,9 @@
       <i class="material-icons right">keyboard_arrow_down</i>
     </div>
     <div class="collapsible-body">
-      <a class="btn waves-effect waves-light submit-btn" (click)="businessChallengeRegister()">Register</a>
+      More details on the Business Challenge and Registration will be avliable closer
```
Comment (stan29308) [inline user-registration-app/src/app/views/user-registration-view/user-registration-view.component.html]: Available*

File: website/index.html [hunk 2]
```diff
@@ -861,33 +861,23 @@
...
+    <div class="row gold" style="padding-top: 2%">
+      <div class="col-sm-6">
+      <!-- Geico -->
+        <a href="https://www.geico.com/" target="_blank">
+          <img src="assets/images/sponsors/Geico-day.svg" alt="Vertex" class="img-fluid Geico-night">
+        </a>
+      </div>
+      <div class="col-sm-6">
+        <!-- GCP -->
+        <a href="https://www.geico.com/" target="_blank">
```
Comment (stan29308) [inline website/index.html]: Should this be going to the GCP website?

File: website/index.html [hunk 3]
```diff
@@ -861,33 +861,23 @@
...
+          <img src="assets/images/sponsors/gcp_F19.svg" alt="Vertex" class="img-fluid Geico-night">
```
Comment (stan29308) [inline website/index.html]: Class name is also still set to Geico-night

File: website/index.html [hunk 4]
```diff
@@ -1003,35 +979,30 @@
           <!--<img src="assets/images/sponsors/Dicks-day.svg" alt="Vertex" class="img-fluid Dicks-day">-->
         <!--</a>-->
       <!--</div>-->
+    <!--</div>-->
+    <!-- Custom $1000 -->
+    <div class="row basic d-flex justify-content-center">
+      <div class="col-sm-2">
+        <!-- ICS -->
+        <a href="https://lionlaunchpad.psu.edu/" target=_blank>
```
Comment (stan29308) [inline website/index.html]: Same with this one, link and class name

File: website/index.html [hunk 5]
```diff
@@ -861,33 +861,23 @@
...
-               <!--class="img-fluid oracle-day">-->
-        <!--</a>-->
-      <!--</div>-->
-    <!--</div>-->
+    <!-- Gold Sponsors -->
+    <div class="row gold" style="padding-top: 2%">
+      <div class="col-sm-6">
+      <!-- Geico -->
+        <a href="https://www.geico.com/" target="_blank">
+          <img src="assets/images/sponsors/Geico-day.svg" alt="Vertex" class="img-fluid Geico-night">
```
[the remark this answers is not in the document]
Comment (jmm8046) [reply website/index.html]: done

File: website/index.html [hunk 6]
```diff
@@ -169,10 +169,10 @@
     <!-- MLH Badge -->
     <div class="white">
       <a id="mlh-trust-badge"
-         href="https://mlh.io/seasons/na-2019/events?utm_source=na-2019&utm_medium=TrustBadge&utm_campaign=na-2019&utm_content=white"
+         href="https://mlh.io/seasons/na-2020/events?utm_source=na-hackathon&utm_medium=TrustBadge&utm_campaign=2020-season&utm_content=white"
          target="_blank"><img
-          src="https://s3.amazonaws.com/logged-assets/trust-badge/2019/mlh-trust-badge-2019-white.svg"
-          alt="Major League Hacking 2019 Hackathon Season" style="width:100%"></a>
+              src="https://s3.amazonaws.com/logged-assets/trust-badge/2020/mlh-trust-badge-2020-white.svg"
```
[the remark this answers is not in the document]
Comment (jmm8046) [reply website/index.html]: done

File: website/index.html (hunk 4 above)
Comment (jmm8046) [reply website/index.html]: done

File: website/index.html (hunk 3 above)
Comment (jmm8046) [reply website/index.html]: done

File: website/index.html (hunk 2 above)
Comment (jmm8046) [reply website/index.html]: done

File: user-registration-app/src/app/views/user-registration-view/user-registration-view.component.html (hunk 1 above)
Comment (jmm8046) [reply user-registration-app/src/app/views/user-registration-view/user-registration-view.component.html]: lmao done

**Pull request merged**